application/app/services/upload/multipart_http_ruby_uploader.rb

```diff
@@ -99,9 +99,9 @@ def report_progress
           file: @file_path,
           total: @total_size,
           uploaded: @uploaded,
-          percent: ((@uploaded.to_f / @total_size) * 100).round(2)
+          progress: [ (@uploaded.to_f / @total_size * 100).to_i, 100 ].min
```

Comment:
should progress be calculated in a helper when rendering the view?
it feels a bit unnecessary to calculate all the time, one use it only when displaying.